### DISCUSSION TITLE
Will Jellyfin-Vue have support for multiple servers / switching servers + accounts?

### DISCUSSION THREAD

**Discussion by snvoid** [jellyfin/jellyfin-vue#2145] [Questions] [answered]
The official (regular) jellyfin web client / desktop client support multiple servers and there is an option to switch servers. Will this feature be supported in Jellyfin Vue as well? Thanks

**Comment by ferferga** [comment, marked as answer]
Hello

Unless I'm misunderstanding you, this is already possible: log out and in the login form you'll have the option to change between servers.

If my memories doesn't fail me, this is the same behaviour as the current Web client, but the current web client has a dedicated "Change server" button (that under the hood does the same behaviour, logout the current user when changing servers)

The architecture of Jellyfin Vue is already ready for seamlessly handling multiple sessions, but the UI is not there yet.
